os/board/imxrt1020-evk/src/imxrt_bringup.c

```diff
@@ -394,7 +394,13 @@ int imxrt_bringup(void)
 	}
 	#endif
 #endif
-
+#ifdef CONFIG_VIDEO_NULL
+	ret = video_null_initialize("/dev/video0");
```

Comment:
null driver is virtual driver, so it is not related to any board.
I think we can initialize it in os_start.
@sunghan-chang How do you think?

Reply:
moved the video_null initialize code to "up_initialize" which would be common for all boards/chip